src/library_gl.js

```diff
@@ -3951,6 +3951,8 @@ var LibraryGL = {
 #endif
     GLctx['bindVertexArray'](GL.vaos[vao]);
 #endif
+    var ibo = GLctx.getParameter(GLctx.ELEMENT_ARRAY_BUFFER_BINDING);
+    GL.currElementArrayBuffer = ibo ? (ibo.name | 0) : 0;
```

Comment:
These two lines need to be gated behind a `#if USES_GL_EMULATION` block, since `GL.currElementArrayBuffer` is not tracked if `USES_GL_EMULATION` is not set (and `GLctx.getParameter` is extremely costly)